Src/BootCamp.Chapter/Computer/Motherboard.cs

```diff
@@ -2,5 +2,11 @@
 {
     public class Motherboard
     {
+        private string _motherboard;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_motherboard' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
done